packages/connect/src/connect-error.ts

```diff
@@ -132,7 +132,7 @@ export class ConnectError extends Error {
     return new ConnectError(String(reason), code, undefined, undefined, reason);
   }
 
-  static [Symbol.hasInstance](v: unknown): boolean {
+  static override [Symbol.hasInstance](v: unknown): boolean {
```

Comment:
This was added in 5.6: https://devblogs.microsoft.com/typescript/announcing-typescript-5-6/#correct-override-checks-on-computed-properties